src/store/getters.ts

```diff
@@ -1076,3 +1065,74 @@ export const getStaffPayment = (state: StateIF): StaffPaymentIF => {
 export const getFolioNumber = (state: StateIF): string => {
   return state.stateModel.newRequestModel.folioNumber
 }
+/** Name Check Getters
+ * TODO: move existing getters used only for name check above to here
+ * TODO: eventually move this all out of vuex (if we refactor to composition api)
```

Comment:
What would the scope of this look like?? Pretty large task?

Reply:
I would say probably a day or 2ish if the first batch of initial composition-api setup stuff is done first. Almost none of the name-check stuff is needed in the vuex global store and it would probably cut the variables/functions in getters/actions/mutations/state in half almost

Reply:
That would be nice! I need to get some more experience with composition-api ...😁